dashboard/app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -206,17 +221,25 @@ def silent_takeover(oauth_user, auth_hash)
         # This should never happen if other logic is working correctly, so notify
         Honeybadger.notify(
           error_class: 'Failed to create AuthenticationOption during silent takeover',
-          error_message: "Could not create AuthenticationOption during silent takeover for user with email #{oauth_user.email}"
+          error_message: "Could not create AuthenticationOption during silent takeover for user with email #{lookup_email}"
         )
       end
     else
-      @user.provider = oauth_user.provider
-      @user.uid = oauth_user.uid
-      @user.oauth_refresh_token = oauth_user.oauth_refresh_token
-      @user.oauth_token = oauth_user.oauth_token
-      @user.oauth_token_expiration = oauth_user.oauth_token_expiration
```

Comment:
How.... how did this work before?  Were we never saving the user after assigning these properties?

Reply:
i'm honestly not sure, but your point makes sense. eric and i ran into this a few days ago and didn't understand, so i figured we should update it